frontend/src/metabase-lib/Question.ts

```diff
@@ -486,7 +486,16 @@ class QuestionInner {
 
   supportsImplicitActions(): boolean {
     const query = this.query();
-    return query instanceof StructuredQuery && !query.hasAnyClauses();
+
+    // we want to check the metadata for the underlying table, not the model
+    const tableId = query.tableId();
+    const table = this.metadata().tables?.[tableId];
+    const hasSinglePk =
+      table?.fields?.filter(field => field.isPK())?.length === 1;
```

Comment:
@alxnddr Is this the right way to get at the underlying table's metadata? it works, but it feels a little dirty.

Reply:
I think I found a better way with `query.sourceTable()`